task_1/birthday_cake/main.go

```diff
@@ -12,9 +14,26 @@ package main
 */
 
 func cake(x int, y string) string {
-	panic("Implement me!")
+	var total int
+	for i, char := range y {
+		if i%2 == 0 {
+			total = total + int(char)
```

Comment:
let's use strconv for converting strings to int

Reply:
fix